src/main/java/com/mars/review/ui/ReviewController.java

```diff
@@ -0,0 +1,36 @@
+package com.mars.review.ui;
+
+import com.mars.review.application.ReviewService;
+import com.mars.review.dto.ReviewRequest;
+import com.mars.review.dto.ReviewResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping(value = "/api/review")
+public class ReviewController {
+    private final ReviewService reviewService;
+
+    @Autowired
+    public ReviewController(ReviewService reviewService) {
+        this.reviewService = reviewService;
+    }
```

Comment:
이미 생성자 주입을 명시해줬기 때문에 `@Autowired`는 필요없을 것 같습니다?!
```suggestion
    public ReviewController(ReviewService reviewService) {
        this.reviewService = reviewService;
    }
```

Reply:
넵!